cpp/msbuild/ice.proj

```diff
@@ -13,7 +13,6 @@
         <!-- The version for all NuGet packages built with this distribution. -->
         <!-- Defaults to IcePackageVersion, which is set in ice.version.props imported above. -->
         <PackageVersion>$(IcePackageVersion)</PackageVersion>
-        <VCTargetsPath Condition="'$(VCTargetsPath)' == ''">C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0</VCTargetsPath>
```

Comment:
This was required with some older Visual Studio versions, but not with the supported versions in 3.8.